engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -74,12 +74,22 @@ import scala.concurrent.duration.DurationInt
 class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
 
   private val log = LoggerFactory.getLogger(this.getClass)
+  private val telemetryLog =
+    LoggerFactory.getLogger("org.enso.telemetry.languageserver.boot.MainModule")
   log.debug(
     "Initializing main module of the Language Server from [{}, {}, {}]",
     BuildVersion.currentEdition,
     serverConfig,
     logLevel
   )
+  telemetryLog.trace(
+    "Initializing main module of the Language Server; edition, graal_version, enso_version, is_release, AOT",
```

Comment:
Doesn't [the specification](https://github.com/enso-org/enso/issues/12464#issuecomment-2713545076) suggest to use `{}` as placeholders? E.g. this should be:
```suggestion
    "Initializing main module of the Language Server: edition={}, graal_version={}, enso_version={}, is_release={}, AOT={}",
```
otherwise, the message wouldn't be properly formatted when printed to console with `--log-level trace`

Reply:
Done in e5267d96f432735a79e16ab15b6cea35295d9505